src/Microsoft.AspNetCore.Blazor.Cli/Microsoft.AspNetCore.Blazor.Cli.csproj

```diff
@@ -11,6 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.CommandLineUtils" Version="1.1.1" />
```

Comment:
@SteveSandersonMS - note this is using the publicly shipped build of CommandLineUtils. We shipped this on nuget.org (by mistake) in the 1.x releases and we continue to maintain and build it internally (on myget). I'm not aware of a significant delta between 1.x and the current bits - so I think this is fine for now. When/if this code starts building along with the product we should update.

Reply:
it's the same version that you use in the build tools https://github.com/aspnet/Blazor/blob/5f6de9b6f50bdf849e661a4f85ea1400228f7e88/src/Microsoft.AspNetCore.Blazor.BuildTools/Microsoft.AspNetCore.Blazor.BuildTools.csproj#L10